src/SFA.DAS.ApplyService.Data/OrganisationRepository.cs

```diff
@@ -88,18 +83,18 @@ FROM Organisations
             }
         }
 
-        public async Task<Organisation> UpdateOrganisation(Organisation organisation)
+        public async Task UpdateOrganisation(Organisation organisation)
```

Comment:
Wouldn't it be better if this returns a bool or something to flag that an update has been successful?

Also, now that you don't return the Organisation back, there is the potential that the caller will assume that the Organisation that they asked to be updated will have those values.  Properties such as UpdatedBy and UpdatedAt will be out of sync.

Additionally there are several private methods that are updating the Organisation, hence this could exasperate the issue.